core-bundle/contao/templates/modules/mod_navigation.html5

```diff
@@ -4,6 +4,7 @@ $this->wrapperAttributes = $this
     ->attr($this->cssID)
     ->addClass([$this->class, 'block'])
     ->addStyle($this->style)
+    ->setIfExists('aria-label', $this->ariaLabel)
```

Comment:
```suggestion
    ->set('aria-label', $this->ariaLabel ?: $this->trans('MSC.mainNavigation'))
```

wdyt? We could apply the same to the other two modules.

Reply:
Actually never mind, this would cause the issue I mentioned above (having multiple navigation modules with the same `aria-label` on the same page, which is more likely with the regular navigation module).